pkgs/applications/office/kmymoney/default.nix

```diff
@@ -1,20 +1,29 @@
 { stdenv, fetchurl, cmake, kdelibs, automoc4, kdepimlibs, gettext, pkgconfig
 , shared_mime_info, perl, boost, gpgme, gmpxx, libalkimia, libofx, libical
-, doxygen }:
+, oxygen_icons, kactivities, aqbanking, perlPackages, gwenhywfar }:
 
 stdenv.mkDerivation rec {
-  name = "kmymoney-4.6.4";
+  name = "kmymoney-4.7.2";
 
   src = fetchurl {
-    url = "mirror://sourceforge/kmymoney2/${name}.tar.xz";
-    sha256 = "04n0lgi2yrx67bgjzbdbcm10pxs7l53srmp240znzw59njnjyll9";
+    url = http://kde.mirrorcatalogs.com/stable/kmymoney/4.7.2/src/kmymoney-4.7.2.tar.xz;
+    #sha256 = "7749cbae146eb4adf5c92162c841ae321f971c5720bc32d0227a42a4dd4acfc4"; # v. 4.7.1
+    sha256 = "bfb2c29ff30988f46324c2dae197a06b58d07336a1947adc22bcfed3e554393d"; # v. 4.7.2
   };
 
   buildInputs = [ kdepimlibs perl boost gpgme gmpxx libalkimia libofx libical
-                  doxygen ];
-  nativeBuildInputs = [ cmake automoc4 gettext shared_mime_info pkgconfig ];
+                  oxygen_icons kactivities shared_mime_info aqbanking
+                  perlPackages.FinanceQuote gwenhywfar ];
+  nativeBuildInputs = [ cmake automoc4 gettext pkgconfig ]; 
 
-  KDEDIRS = libalkimia;
+  #KDEDIRS = libalkimia;
+  KDEDIRS="$out:${libalkimia}:${gwenhywfar}:${aqbanking}";
+
+  postInstall = ''
+    ln -s ${shared_mime_info}/share/mime/application/* $out/share/mime/application/
```

Comment:
@ttuegel thanks! i'm learning nix (mostly just trying to use nixos) and that's a top tip. I'll have to wrap my head around propagatedUserEnvPkgs at some point, obviously.

Btw I've just looked and yes it seems like it's /nix/store/yqvcp3nq0wgqxydqg3k1mclla54dsy0j-akonadi-1.13.0/share/mime/aliases which conflicts (with some things on my system). Perhaps I'll open a bug report for skrooge and akonadi etc which seem to be in conflict as well.